File: 389. Find the Difference/389.py
class Solution(object):
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        ans = ord(t[0])
        for i in range(len(s)):
            ans ^= ord(s[i])
        for i in range(1, len(t)):
            ans ^= ord(t[i])
        return chr(ans)

File: 389-Find the Difference/389.c
char findTheDifference(char* s, char* t) {
    int i=0, j=1;
    while (s[i]!='\0') t[0] ^= s[i++];
    while (t[j]!='\0') t[0] ^= t[j++];
    return t[0];
}